priv/repo/migrations/20161024204753_remove_package_downloads_pk.exs

```diff
@@ -0,0 +1,84 @@
+defmodule HexWeb.Repo.Migrations.RemovePackageDownloadsPk do
+  use Ecto.Migration
+
+  def up do
```

Comment:
no reason, we simply don't need the pk anymore but it doesn't hurt to keep it around either